feat(flags): Consolidate feature flags pull requests (#12005)
getsentry/sentry-docs: platform-includes/feature-flags/next-steps/_default.mdx

> Please read the note below to ensure that you also complete one additional step.

Reviewer: ```suggestion
Please read the note below and complete one more step:
```

<Alert level="warning" title="Next Steps">

- **Track feature flag evaluations in other parts of your codebase.** If needed, you can set up evaluation tracking for more than one SDK. [Read the docs](/product/explore/feature-flags/#set-up-evaluation-tracking) to learn more. 
- **Set up your change tracking webhook.** In order to take full advantage of the feature flag capabilities Sentry offers, there is an additional setup step needed.  Setting up one or more provider webhooks is needed to enable feature flag change tracking, so that your feature flag provider may communicate feature flag changes to Sentry. Learn how to set this up by [reading the docs](/product/explore/feature-flags/#set-up-change-tracking).

> 

Reviewer: ```suggestion
- **Track feature flag evaluations in other parts of your codebase.** If needed, you can set up evaluation-tracking for [more than one SDK](/product/explore/feature-flags/#set-up-evaluation-tracking).
- **Set up your change-tracking webhook.** In order to take full advantage of Sentry's feature flag capabilities, set up one or more webhook provider to allow your feature flag provider to send change updates to Sentry. Learn [how](/product/explore/feature-flags/#set-up-change-tracking).

```
